### DISCUSSION TITLE
Chia 1.8.2 is not supported on Windows 8.1?

### DISCUSSION THREAD

**Discussion by gooverup** [Chia-Network/chia-blockchain#15707] [Support]
Error when running Chia 1.8.2 on Windows 8.1.
This error appears and the program does not start:

_Chia.exe - Entry Point Not Found.
The procedure entry point DiscardVirtualMemory could not be located in the dynamic link library._
![Chia exe - Entry Point Not Found](https://github.com/Chia-Network/chia-blockchain/assets/9006193/53fac0b0-3265-4321-bb6f-19be614176be)

**Comment by BrandtH22** [comment]
Hey @gooverup, I am sorry for the delayed response with our support channels moving from keybase to discord (https://discord.gg/chia) we have not given enough attention to the github issues but are working to improve our responses on all channels.

This appears to be indicating that there was an installation issue which is preventing access to the Chia.exe script.
Have you tried to uninstall, reboot the computer, then install the latest version (1.8.2)?

If that does not work and if possible (dependent on your device setup and security needs) I would try to install for all users (requires admin approval but installs to the Program Files directory rather than the AppData directory).

Let us know if this works and whether you are able to get it up and running 👍
